plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/GitEventsSubscriber.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.ide.ext.git.client;
+
+import org.eclipse.che.api.git.shared.FileChangedEventDto;
+import org.eclipse.che.api.git.shared.StatusChangedEventDto;
+import org.eclipse.che.api.project.shared.dto.event.GitCheckoutEventDto;
+
+/**
+ * Describes git events.
+ *
+ * @author Mykola Morhun
+ */
+public interface GitEventsSubscriber {
+
+  /** Invoked when a project file was changed */
+  void onFileChanged(String endpointId, FileChangedEventDto dto);
+
+  // TODO change method name or behaviour. It is not git status actually.
```

Comment:
Please add more description about why it is not a git status.

Reply:
Done